test/JDBC/input/functions/BABEL-3953-datetrunc-vu-verify.sql

```diff
@@ -0,0 +1,200 @@
+SELECT * FROM DATETRUNC_vu_prepare_v1
+GO
+
+SELECT * FROM DATETRUNC_vu_prepare_v2
+GO
+
+SELECT * FROM DATETRUNC_vu_prepare_v3
+GO
+
+SELECT * FROM DATETRUNC_vu_prepare_v4
+GO
+
+SELECT * FROM DATETRUNC_vu_prepare_v5
+GO
+
+SELECT * FROM DATETRUNC_vu_prepare_v6
+GO
+
+SELECT * FROM DATETRUNC_vu_prepare_v7
+GO
+
+SELECT * FROM DATETRUNC_vu_prepare_v8
+GO
+
+SELECT * FROM DATETRUNC_vu_prepare_v9
+GO
+
+SELECT * FROM DATETRUNC_vu_prepare_v10
+GO
+
+SELECT * FROM DATETRUNC_vu_prepare_v11
+GO
+
+EXEC BABEL_3953_vu_prepare_p1
+GO
+
+EXEC BABEL_3953_vu_prepare_p2
+GO
+
+EXEC BABEL_3953_vu_prepare_p3
+GO
+
+EXEC BABEL_3953_vu_prepare_p4
+GO
+
+EXEC BABEL_3953_vu_prepare_p5
+GO
+
+EXEC BABEL_3953_vu_prepare_p6
+GO
+
+SELECT BABEL_3953_vu_prepare_f1()
+GO
+
+SELECT BABEL_3953_vu_prepare_f2()
+GO
+
+SELECT BABEL_3953_vu_prepare_f3()
+GO
+
+select datetrunc(null, CAST('2020-01-01' as date)) as dt1
+GO
+select datetrunc(null, null) as dt1
+GO
+select datetrunc(null, 'NULL') as dt1
+GO
+select datetrunc('NULL', null) as dt1
+GO
+select datetrunc('NULL', 'NULL') as dt1
+GO
+select datetrunc(year, null) as dt3
+GO
+select datetrunc(years, null) as dt4
+GO
+select datetrunc(nanosecond ,null) as dt5
```

Comment:
Add the testcase such as datetrunc('year',CAST('2020-01-01' as date)) and similar ones if not added already.